marklogic-data-hub-central/ui/src/components/entities/matching/multi-slider/multi-slider.scss

```diff
@@ -178,6 +178,9 @@
               width: 165px;
               overflow: hidden;
             }
+            .close {
+                margin-top: 7px;
```

Comment:
Let's change to the following so the close icon is placed in the upper-right corner as is currently:
```
margin-top: 1px;
```